build.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Copyright 2020 Red Hat, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# this is improper - we need to start using tags in GitHub properly
+version=0.5
+
+buildtime=$(date)
+branch=$(git rev-parse --abbrev-ref HEAD)
+commit=$(git rev-parse HEAD)
+
+go build -ldflags="-X 'main.BuildTime=$buildtime' -X 'main.BuildVersion=$version' -X 'main.BuildBranch=$branch' -X 'main.BuildCommit=$commit'"
+exit $?
```

Comment:
How will this work with the current deployment process?

Reply:
Mmmm good point. The Travis and Codecov apparently have git in their environments, but IDK if it will be the case on Jenkins to handle the stable branch...

Reply:
@JoseLSegura it should work fine. Jenkins doesn't build the code for that repo. It operates images.

Reply:
I was concerned about the container image generation process, but I have reviewed the Dockerfile and I saw that we already need `git` to generate the container, so no problems at all